projects/extension/sql/idempotent/905-semantic-catalog-render.sql

```diff
@@ -290,3 +290,17 @@ as $func$
 $func$ language sql stable security invoker
 set search_path to pg_catalog, pg_temp
 ;
+
+-------------------------------------------------------------------------------
+-- render_obj_sample
+create or replace function ai.render_obj_sample
```

Comment:
I think "render_table_sample" or just "render_sample" might be better understood by users

Reply:
Done in 6be2c632acad662561e3846611ec93f91ca7a313